Source/interactivity_manager_impl.cpp

```diff
@@ -1171,28 +1188,28 @@ interactivity_manager_impl::on_socket_message_received(
     m_unhandledFromService.push(rpcMessage);
 }
 
-void interactivity_manager_impl::process_reply(const web::json::value& jsonReply)
+bool interactivity_manager_impl::process_reply(const web::json::value& jsonReply)
 {
     LOGS_INFO << "Received a reply from the service";
-
+    bool messageWasProcessed = true;
     try
     {
         if (jsonReply.has_field(L"id"))
         {
             std::lock_guard<std::recursive_mutex> lock(m_messagesLock);
             std::shared_ptr<interactive_rpc_message> message = remove_awaiting_reply(jsonReply.at(RPC_ID).as_integer());
 
-			if (message == nullptr)
-			{
-				LOGS_ERROR << L"Message does not exist: " << jsonReply.at(RPC_ID).as_string();
-				return;
-			}
+            if (message == nullptr)
+            {
+                LOGS_ERROR << L"Message does not exist: " << jsonReply.at(RPC_ID).as_string();
+                return false;
+            }
 
             if (jsonReply.has_field(RPC_ERROR))
             {
                 LOGS_ERROR << L"Full message for error reply: " << message->m_json.serialize();
                 process_reply_error(jsonReply);
-                return;
+                return false;

```

Comment:
>return false; [](start = 16, length = 13)

not correct - an error reply should not return false, because it is a valid message. maybe process_reply_error should also return a bool here?